styles.css

```diff
@@ -323,17 +273,12 @@ form input {
   cursor: pointer;
 }
 
-.submit:active {
-  background-color: #2230d2;
-  color: #fff;
-}
-
 .works {
   display: grid;
   grid-template-columns: 1fr;
   margin: 0 auto;
-  padding: 0 20px;
-  gap: 8rem;
+  row-gap: 8rem;
+  padding: 0 60px;
 }
```

Comment:
![design](https://user-images.githubusercontent.com/63917725/177222475-694579ec-bdc4-44ed-9565-58f2fd299c4f.PNG)
- [ ] Please make your design responsive by adding width in percentage instead of pixels. With this approach, your design would be responsive. :blush: